src/test/regress/expected/subquery_in_where.out

```diff
@@ -6,7 +6,6 @@ SET search_path TO subquery_in_where, public;
 SET client_min_messages TO DEBUG1;
 --CTEs can be used as a recurring tuple with subqueries in WHERE
 -- prevent PG 11 - PG 12 outputs to diverge
-SET citus.enable_cte_inlining TO false;
 WITH event_id
```

Comment:
This test definitely expects MATERIALIZED cte: `CTEs can be used as a recurring tuple with subqueries in WHERE`